examples/python.KStar/bbkstar.py

```diff
@@ -36,55 +36,60 @@
 
 # how should we compute energies of molecules?
 # (give the complex conf space to the ecalc since it knows about all the templates and degrees of freedom)
-parallelism = osprey.Parallelism(cpuCores=4)
+parallelism = osprey.Parallelism(cpuCores=20)
 minimizingEcalc = osprey.EnergyCalculator(complexConfSpace, ffparams, parallelism=parallelism, isMinimizing=True)
 
 # BBK* needs a rigid energy calculator too, for multi-sequence bounds on K*
 rigidEcalc = osprey.SharedEnergyCalculator(minimizingEcalc, isMinimizing=False)
 
 # how should we define energies of conformations?
 def confEcalcFactory(confSpace, ecalc):
-	eref = osprey.ReferenceEnergies(confSpace, ecalc)
-	return osprey.ConfEnergyCalculator(confSpace, ecalc, referenceEnergies=eref)
+    eref = osprey.ReferenceEnergies(confSpace, ecalc)
+    return osprey.ConfEnergyCalculator(confSpace, ecalc, referenceEnergies=eref)
+
+epsilon = 0.99
 
 # configure BBK*
 bbkstar = osprey.BBKStar(
-	proteinConfSpace,
-	ligandConfSpace,
-	complexConfSpace,
-	numBestSequences=2,
-	epsilon=0.99, # you proabably want something more precise in your real designs
-	writeSequencesToConsole=True,
-	writeSequencesToFile='bbkstar.results.tsv'
+    proteinConfSpace,
+    ligandConfSpace,
+    complexConfSpace,
+    numBestSequences=7,
+    epsilon=0.1, # you proabably want something more precise in your real designs
```

Comment:
The example scripts are designed to just illustrate OSPREY API ideas and finish quickly. Is it really necessary to do precise (and slow) computations to accomplish those goals?